client/data/disputes/reducer.js

```diff
@@ -11,7 +11,14 @@ import { map, keyBy } from 'lodash';
 import TYPES from './action-types';
 import { getResourceId } from 'utils/data';
 
-const defaultState = { byId: {}, queries: {} };
+const defaultState = {
+	byId: {},
+	queries: {},
+	isSavingEvidenceForDispute: {},
+	isUploadingEvidenceForDispute: {},
+	evidenceUploadErrorsForDispute: {},
+	evidenceTransientForDispute: {},
```

Comment:
This is essentially what used to be component-level state, but it's now been moved into the wp.data state tree.